icons/aperture.svg

```diff
@@ -10,10 +10,10 @@
   stroke-linejoin="round"
 >
   <circle cx="12" cy="12" r="10" />
-  <line x1="14.31" x2="20.05" y1="8" y2="17.94" />
-  <line x1="9.69" x2="21.17" y1="8" y2="8" />
-  <line x1="7.38" x2="13.12" y1="12" y2="2.06" />
-  <line x1="9.69" x2="3.95" y1="16" y2="6.06" />
-  <line x1="14.31" x2="2.83" y1="16" y2="16" />
-  <line x1="16.62" x2="10.88" y1="12" y2="21.94" />
+  <path d="m7.4 12 5.7-9.9" />
+  <path d="M9.7 8h11.5" />
+  <path d="m14.3 8 5.8 9.9" />
+  <path d="m16.6 12-5.7 9.9" />
+  <path d="M14.3 16H2.8" />
+  <path d="M9.7 16 4 6.1" />
```

Comment:
This one's also overoptimised and the lines and the circles don't align anymore:
```suggestion
  <path d="m14.31 8 5.74 9.94" />
  <path d="M9.69 8h11.48" />
  <path d="m7.38 12 5.74-9.94" />
  <path d="M9.69 16 3.95 6.06" />
  <path d="M14.31 16H2.83" />
  <path d="m16.62 12-5.74 9.94" />
```